.changeset/twenty-pans-agree.md

```diff
@@ -0,0 +1,5 @@
+---
+'@astrojs/netlify': minor
+---
+
+Add on-demand builders Netlify functions
```

Comment:
```suggestion
Add `builders` config option for Netlify On-demand Builders.
```
Just a suggestion! Netlify does appear to capitalize both "On" and "Builders". And, the change is adding this new config option to the adapter, so this is explicit about that. (Don't think you need to say that it's only for Netlify Functions, not Edge, in this changelog description. That's for the README.)